client/http/http_test.go

```diff
@@ -44,9 +51,14 @@ func TestHTTPClient(t *testing.T) {
 }
 
 func TestHTTPGetLatest(t *testing.T) {
-	addr, chainInfo, cancel, _ := mock.NewMockHTTPPublicServer(t, false)
+	addr, chainInfo, cancel, _ := mock.NewMockHTTPPublicServer(t, false, utils.PrevSigDecoupling())
 	defer cancel()
 
+	err := WaitServerToBeReady(t, addr)
```

Comment:
Just a note as future reference: this for example could be addressed in a first separate PR, as I dont think that matters to adding unchained or not.